core/src/saros/negotiation/OutgoingSessionNegotiation.java

```diff
@@ -85,9 +85,11 @@ public OutgoingSessionNegotiation( //
 
   @Override
   protected void executeCancellation() {
-    final User user = sarosSession.getUser(getPeer());
+    if (getPeer().isResourceQualifiedJID()) {
+      User user = sarosSession.getUser(getPeer());
```

Comment:
What is the point of removing the final keyword ? I normally prefix every variable with final if this variable should not be changed later.

Reply:
No offense but I think it is just clutter in this case, because there is no further meaning in marking a user explicitly final within 5 lines of code.